app/src/main/res/drawable/ic_quest_cow.xml

```diff
@@ -5,20 +5,24 @@
     android:viewportHeight="128">
```

Comment:
What did you do here?

Reply:
no idea at all what happened here, I was importing drawable that I see was apparently already present but not detected - maybe due to some che issues